package.json

```diff
@@ -34,6 +34,7 @@
     "yargs": "^4.7.1"
   },
   "dependencies": {
+    "azure-devops-node-api": "^10.2.1",
```

Comment:
Why this (and corresponding `package-lock.json`) change?

Reply:
removed